### PR TITLE
Drop shebangs from scripts and add a section to ABOUT

### LINKED ISSUE #158: shebang is broken
```bash
» head -n1 leap.awk
#!/usr/bin/env gawk -f
» chmod 755 leap.awk
» ./leap.awk <<< 2
/usr/bin/env: ‘gawk -f’: No such file or directory
/usr/bin/env: use -[v]S to pass options in shebang lines

# Modified shebang:
» head -n1 leap.awk
#!/usr/bin/env -S gawk -f
» ./leap.awk <<< 2
false
```

The shebang `#!/usr/bin/env gawk -f` is completely broken. "According to POSIX, the hashbang can have one argument." (see [SO](https://stackoverflow.com/questions/733824/how-to-run-a-shell-script-on-a-unix-console-or-mac-terminal/733901#733901)).

Our options here are (1) hard code a `/usr/bin/gawk` path or (2) rely on GNU `env` and `-S` or (3) drop the shebang and not rely on it.

See also [Wikipedia](https://en.wikipedia.org/wiki/Shebang_(Unix)#Character_interpretation) and [netbsd mail list](http://mail-index.netbsd.org/netbsd-users/2008/11/09/msg002388.html)

@glennj Thoughts?

### PR DESCRIPTION
**Pull request by IsaacG**
Closes #158

### CODE REVIEW

File: docs/ABOUT.md [hunk 1]
```diff
@@ -38,6 +38,17 @@ Another common implementation is `mawk`, which is based on a bytecode interprete
 This track will use `gawk`, GNU's awk implementation.
 `gawk` may not have the raw speed of `mawk`, but it is very feature-packed, well documented, and has some handy command-line options.
 
+### shebang
```
Comment (glennj) [inline docs/ABOUT.md:41]: Capitalize please

File: docs/ABOUT.md [hunk 2]
```diff
@@ -38,6 +38,17 @@ Another common implementation is `mawk`, which is based on a bytecode interprete
...
+
+Depending on the platform, awk scripts can be executed by adding a "shebang" line to the top of the file.
```
Comment (glennj) [inline docs/ABOUT.md:43]: For the uninitiated perhaps a link to https://en.wikipedia.org/wiki/Shebang_(Unix)

File: docs/ABOUT.md [hunk 3]
```diff
@@ -38,6 +38,17 @@ Another common implementation is `mawk`, which is based on a bytecode interprete
...
+One most systems, one of the following two shebangs ought to work.
+However, since there is no one shebang which works for all systems, we have choosen to omit shebangs.
+
+```bash
+#!/usr/bin/env -S gawk -f
+#!/usr/bin/gawk -f
+```
+
```
Comment (glennj) [inline docs/ABOUT.md:51]: Even an invalid shebang like `#!gawk` can act a hint to your editor to apply the proper syntax highlighting.

**Review by glennj — APPROVED** [review]

**Comment by glennj** [comment]
Thanks for taking on this tedious task.

File: docs/ABOUT.md (hunk 1 above)
Comment (IsaacG) [reply docs/ABOUT.md]: Done.

**Review by glennj — APPROVED** [review]

**Pull request merged**